src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -2081,10 +2081,11 @@ public boolean onRightClick(ItemUseEvent e, final Player p, ItemStack item) {
 
 		@SuppressWarnings("unchecked")
 		final String[] explosiveblacklist = Slimefun.getItemValue("EXPLOSIVE_PICKAXE", "unbreakable-blocks") != null ? ((List<String>) Slimefun.getItemValue("EXPLOSIVE_PICKAXE", "unbreakable-blocks")).toArray(new String[((List<String>) Slimefun.getItemValue("EXPLOSIVE_PICKAXE", "unbreakable-blocks")).size()]): new String[] {"BEDROCK", "BARRIER", "COMMAND", "COMMAND_CHAIN", "COMMAND_REPEATING"};
+		final boolean damageOnUse = Boolean.TRUE.equals(((Boolean) Slimefun.getItemValue("EXPLOSIVE_PICKAXE", "damage-on-use")));
```

Comment:
You probably should rather just do Boolean.parseBoolean(string) to get a boolean rather than this comparison...

Reply:
I thought of that but the problem I faced was that `getItemValue()` for this value either returns a Boolean or `null`, and not a String. Unless I'm missing something here, there's no way to convert it to the String that Boolean.parseBoolean() wants without doing some extra checks (is it null, if not, make string).